index.js

```diff
@@ -111,19 +119,19 @@ function* run() {
     if (result.set) {
       // There's already a replica set config, so don't initiate
       yield rs.waitForPrimary();
-    } else {
-      // First time starting up, need to create a replica set config
-      for (const manager of rs.managers) {
-        yield manager.stop();
+      } else {
+        // First time starting up, need to create a replica set config
+        for (const manager of rs.managers) {
+          yield manager.stop();
```

Comment:
What's up with this extra indent? Not a big deal, just curious

Reply:
I am not sure I messed up the indents so I was trying to get them back to some good look! Thanks.